manim/mobject/coordinate_systems.py

```diff
@@ -1900,8 +1963,8 @@ def construct(self):
             self._origin_shift([self.x_axis.x_min, self.x_axis.x_max]),
         )
         result = np.array(origin)
-        for axis, coord in zip(self.get_axes(), coords):
-            result += axis.number_to_point(coord) - origin
+        for i, (axis, coord) in enumerate(zip(self.get_axes(), coords)):
+            result[i] += axis.number_to_point(coord)[i] - origin[i]
```

Comment:
The old version seems like it could handle the case of rotated axes as well, which the new version can't (?) – is there really an advantage to changing this?

Reply:
I think he had planned to go back to the previous behavior as I was discussing with him yesterday.